### ISSUE TITLE
Performance gapwhen fine-tuning on CSL-Daily with provided checkpoints

### ISSUE THREAD

**Issue by XieYingchun** [ZechengLi19/Uni-Sign#27]
Description First of all, thank you for your inspiring work and for releasing the code/checkpoints.

I am trying to reproduce the results on the CSL-Daily dataset using the provided pre-trained weights and the default configuration. However, I noticed a consistent performance gap of about 1.0 point compared to the results reported in the paper.

Experiment Environment

GPU: NVIDIA RTX A6000

Config: Default parameters (as provided in the repository)

Task: Fine-tuning on CSL-Daily using the author's pre-trained weights.

<img width="1912" height="330" alt="Image" src="https://github.com/user-attachments/assets/96a1e6d4-7f97-4d50-b1d7-1ebfecb201e0" />

Questions I followed the default scripts, but I'm struggling to close this ~1% gap. I would appreciate it if you could provide some insights on the following:

Hyperparameters: Did you use any specific learning rate scheduler or warmup strategy that might differ from the default config when fine-tuning on A6000?

Batch Size: Since I am using an A6000 (48GB), does the batch size significantly affect the final performance for this model? What was the exact batch size used for the paper results?

Environment: Could the version of PyTorch or CUDA lead to this slight degradation?

Thank you for your time and help!

**Comment by ZechengLi19** [comment]
@XieYingchun Thank you very much for your interest in Uni-Sign.

In my experiments, all hyperparameters (including the batch size) are kept strictly consistent with those reported in the paper and provided in the code repository. Therefore, the observed discrepancy may potentially stem from differences in GPU models or the Python/software environment.

In general, achieving a one-point performance improvement is relatively straightforward. You may consider fine-tuning the model using fewer GPUs while keeping all other hyperparameters unchanged. In many cases, this setup can lead to additional performance gains. However, in Uni-Sign, I intentionally did not adopt this strategy in order to maintain consistent hyperparameter settings across all training stages.

If you are still concerned about whether the original bash scripts can reproduce the results reported in the paper, I would suggest trying to run the experiments on a different GPU setup (like A40 or RTX3090).
